.github/ISSUE_TEMPLATE/RFC.md

```diff
@@ -0,0 +1,18 @@
+---
+name: RFC for new interface
+about: Use this template to request new functionality or change behavior of the library
+title: ''
+labels: 'RFC'
+assignees: ''
+---
+
+# Summary
+Include a short summary of the request. Sections below provide guidance on
+what factors are considered important. Describe how new interface will meet [library functionality guidelines](https://github.com/oneapi-src/oneMKL/blob/master/CONTRIBUTING.md#library-functionality-guidelines).
+
+# Problem statement
+Describe the problem you are trying to solve with reasonable level of details. 
+
+# Details
```

Comment:
What kind of detail are we expecting if the RFC is about library behavior change? (or anything besides new functionality)

Reply:
Templates are created to help to structure information for the most common requests. I would expect that interfaces will be the major part of requests. Submitter don't have to use template and they can select empty template to fill in. if we see there are many requests coming for some other area, we always can add template later.